Graphs/BellmanFord.js

```diff
@@ -0,0 +1,51 @@
+//The Bellman–Ford algorithm is an algorithm that computes shortest paths 
```

Comment:
Please provide links to the algorithm's description. See https://github.com/TheAlgorithms/Javascript/issues/630 for more details for commenting.

Reply:
Sure, I'll make the changes. Thanks!